doc/source/userguide.rst

```diff
@@ -220,6 +284,10 @@ This backend can be specified via::
 
     scattering = Scattering2D(J=2, shape=(32, 32), backend='torch_skcuda')
 
+Each of the other frontends currently only has a single backend, which is the
+default. Work is currently underway, however, to extend some of these frontends
```

Comment:
"Work is currently underway to extend some of these frontends" OR "However, work is currently underway to extend some of these frontends"